### PR TITLE
Ignore routing specs for DescribeClass cop

### PR DESCRIPTION
**Pull request by conzett**
In the same vein as the request and feature specs: https://www.relishapp.com/rspec/rspec-rails/docs/routing-specs

I can squash if you would be interested in merging this :thumbsup:

### CODE REVIEW

File: spec/rubocop/cop/rspec/describe_class_spec.rb [hunk 1]
```diff
@@ -36,6 +36,11 @@
     inspect_source(cop, "describe 'my new feature', type: :feature do; end")
     expect(cop.offenses).to be_empty
   end
+  
+  it 'ignores routing specs' do
+    inspect_source(cop, "describe 'my new feature', type: :routing do; end")
```
Comment (mikegee) [inline spec/rubocop/cop/rspec/describe_class_spec.rb]: mebbe describe 'my new route'?

**Comment by geniou** [comment]
@conzett could you please
- change the test corresponding to the comment of @mikegee
- squash the two commits

It would also be nice if you could add a comment to the Changelog with this. Then I would merge it and release a new version. Thanks.

**Comment by nijikon** [comment]
@conzett Please rebase on top of master

**Comment by nijikon** [comment]
@conzett ping

**Comment by nijikon** [comment]
Merged in 125c567da597b8ebac5e9c8265626906f9d9aefc

**Pull request closed without merging**